remix-solidity/src/compiler/compiler.ts

```diff
@@ -0,0 +1,392 @@
+'use strict'
+
+var solc = require('solc/wrapper')
+var solcABI = require('solc/abi')
+var webworkify = require('webworkify')
+import compilerInput from './compiler-input'
+var remixLib = require('remix-lib')
+var EventManager = remixLib.EventManager
+var txHelper = require('./txHelper')
+import { Source, SourceWithTarget, EVMVersion, Language, State, CompilationResult } from './types'
+
+/*
+  trigger compilationFinished, compilerLoaded, compilationStarted, compilationDuration
+*/
+export class Compiler {
+  event: any
+  state: State
+  handleImportCall: any
+
+  constructor (handleImportCall) {
```

Comment:
You can do `constructor (public handleImportCall)` then you don't need line 18 & 36

Reply:
cool thanks

Reply:
done